container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/github_auth_storage_creator/github_auth_storage_creator.go

```diff
@@ -28,23 +28,18 @@ const (
 	authStorageCreationCmdMaxRetries     = 2
 	authStorageCreationCmdDelayInRetries = 200 * time.Millisecond
 
+	githubAuthTokenFilePath = "token.txt"
+
 	sleepSeconds = 1800
 )
 
-type GitHubAuthStorageCreator struct {
```

Comment:
remove the object here in favor of functions, as object wasn't providing any benefit